src/drivers/imu/adis16448/adis16448.cpp

```diff
@@ -994,16 +982,16 @@ ADIS16448::ioctl(struct file *filp, int cmd, unsigned long arg)
 					bool want_start = (_call_interval == 0);
 
 					/* convert hz to hrt interval via microseconds */
-					unsigned ticks = 1000000 / arg;
+					unsigned interval = 1000000 / arg;
 
 					/* check against maximum sane rate */
-					if (ticks < 1000) {
+					if (interval < 1000) {
 						return -EINVAL;
 					}
 
 					// adjust filters
 					float cutoff_freq_hz = _accel_filter_x.get_cutoff_freq();
-					float sample_rate = 1.0e6f / ticks;
+					float sample_rate = 1.0e6f / interval;
```

Comment:
You could use the `USEC_PER_SEC` #define here if you want.

Reply:
Let's try it to use the time_literals everywhere. https://github.com/PX4/Firmware/blob/ed8c6019d564c1c8c93cf47a5bfb7a73772fd89f/src/drivers/drv_hrt.h#L200-L223